### PR TITLE
*Add user, media, and comment support to REST API

### PR DESCRIPTION
**Pull request by pcfreak30**
*If $object_type is of type media, change to attachment as thats the identifier in REST

### CODE REVIEW

**Comment by sc0ttkclark** [comment]
Can you pull the latest and resolve conflicts?

**Comment by sc0ttkclark** [comment]
Needs conflict resolution

**Comment by pcfreak30** [comment]
Working on this slowly.

**Comment by pcfreak30** [comment]
This has been done.

File: classes/PodsRESTHandlers.php [hunk 1]
```diff
@@ -65,8 +66,16 @@ public static function get_handler( $object, $field_name, $request ) {
...
-
+		
+		if ( empty( $pod_name ) && 'attachment' == $object_type ) {
+			$pod_name = 'media';
+		}
 		$id  = pods_v( 'id', $object );
+
+		if ( empty( $id ) ) {
+			$id = pods_v( 'ID', $object );
+		}
```
Comment (sc0ttkclark) [inline classes/PodsRESTHandlers.php]: There's some more cases to cover but I'll make this more dynamic when I get a chance.

**Pull request merged**